tests/console/rsync.pm

```diff
@@ -19,11 +19,33 @@ use warnings;
 use testapi;
 use utils;
 use version_utils qw(is_opensuse is_sle is_jeos);
+use Utils::Backends qw(is_svirt_except_s390x);
 
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
     # try to install rsync if the test does not run on JeOS
+    # for the case that openssh is not installed
+    zypper_call("install -y openssh");
+    # create ssh key
+    assert_script_run("mkdir /root/.ssh/");
+    assert_script_run("ssh-keygen -t rsa -P '' -C 'root\@localhost' -f ~/.ssh/id_rsa");
+    assert_script_run("systemctl enable sshd");
+    assert_script_run("systemctl start sshd");
+    assert_script_run("systemctl status sshd");
+
+    # workaround for network issue on svirt backend, this can be removed when PR#15685 got merged
+    # or bsc#1203626 got resolved
+    if (is_svirt_except_s390x) {
+        select_console("root-console");
+        enter_cmd("cd /var/lib/wicked");
+        my @files = qw(duid.xml lease-eth0-dhcp-ipv4.xml lease-eth0-dhcp-ipv6.xml);
+        enter_cmd("for i in \@files; do echo '' > /var/lib/wicked/\@files; done");
+        enter_cmd("cat /var/lib/wicked/\@files");
+        assert_script_run('systemctl restart network');
+        assert_script_run('systemctl status network');
+    }
```

Comment:
My review from 10 days ago still stands: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15644#pullrequestreview-1134099226

Reply:
In that case we can remove this block. Right?

Reply:
yes, I think yes. no blocks, just other workaround.